Update monitoring.html.md (#1807)
hashicorp/terraform-website: content/source/docs/enterprise/admin/monitoring.html.md

## Internal Monitoring

> Beginning in version 201811-1, Terraform Enterprise includes internal monitoring of critical application metrics using a statsd collector. If the Terraform Enterprise instance is already running a collector on the same port, Terraform Enterprise may not start up correctly due to the conflict, and logs will indicate:

Reviewer: I'm just editing for flow. Let me know if you don't like this or it materially changes the meaning! 

```suggestion
Beginning in version 201811-1, Terraform Enterprise uses a statsd collector to monitor critical application metrics. Terraform Enterprise produces an error during start up if the instance is already running a collector on the same port:
```

Author: Thank you! I think it is best to have @sudomateo and @justinretzolk across this.
I am not certain about the intent of this particular section. The PR was only aimed to clarify that the stats are not included in the support bundle.

Author: Probably inaccurate to use the term clarify, it is to remove the ambiguity and misunderstanding.

Reviewer 2: I like the suggestion!

Reviewer: @hashitop - now that Matthew has weighed in on this, do you feel comfortable making this change? :-)
